daemon/archive.go

```diff
@@ -277,7 +277,7 @@ func (daemon *Daemon) containerExtractToDir(container *container.Container, path
 
 	options := daemon.defaultTarCopyOptions(noOverwriteDirNonDir)
 
-	if copyUIDGID {
+	if !copyUIDGID {
```

Comment:
oh, need to remove this one as that's part of https://github.com/moby/moby/pull/34099, so don't merge as-is